src/main/java/org/spongepowered/common/mixin/core/world/MixinWorld.java

```diff
@@ -433,6 +443,11 @@ public Entity createEntity(EntityType type, Vector3d position) throws IllegalArg
             }
         }*/
 
+        if (naturally && entity instanceof EntityLiving) {
+            // Adding the default equipment
+            ((EntityLiving)entity).onInitialSpawn(world.getDifficultyForLocation(new BlockPos((net.minecraft.entity.Entity) entity)), null);
```

Comment:
Need some proper spacing. Also, why are you using the entity constructor when you already have the `x`, `y`, and `z`?